packages/teleport/src/Databases/Databases.tsx

```diff
@@ -58,11 +58,13 @@ export function Databases(props: State) {
     <FeatureBox>
       <FeatureHeader alignItems="center" justifyContent="space-between">
         <FeatureHeaderTitle>Databases</FeatureHeaderTitle>
-        <ButtonAdd
-          isLeafCluster={isLeafCluster}
-          canCreate={canCreate}
-          onClick={showAddDialog}
-        />
+        {!hasDatabases && attempt.status !== 'processing' && (
```

Comment:
hm, i need clarification on when we want to show the button? i thought it was if user has database (successful and non-empty state). Do we want to show the button when it errors out too?